Makefile.docker

```diff
@@ -16,7 +16,7 @@ all: docker_build docker_push
 
 docker_build:
 	# Build Docker image ...
-	docker build -q --build-arg strimzi_version=$(RELEASE_VERSION) -t strimzi/$(PROJECT_NAME):latest $(DOCKERFILE_DIR)
+	docker build -q $(DOCKER_BUILD_ARGS) --build-arg strimzi_version=$(RELEASE_VERSION) -t strimzi/$(PROJECT_NAME):latest $(DOCKERFILE_DIR)
```

Comment:
is this `DOCKER_BUILD_ARGS` something new?

Reply:
Yeah, I think I added because it is useful when you need to for rebuild (`--no-cache`).